app/services/solidus_bolt/base.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'httparty'
```

Comment:
Is this required? 🤔

Reply:
I think it is required

If I remove it then in the classes that inherit from this base class, calling `HTTParty.get` for example makes it look for `SolidusBolt::Account::DetectAccount::HTTParty` instead of the `HTTParty` class

Reply:
Oh yes! This is because you added the gem [as dependency](https://github.com/nebulab/solidus_bolt/pull/21/commits/bf2f6a50f3c80af3550a568264b151b2bc002dc5#diff-0fd7ad3190941f3d7db9d5596e43b49dfe18a5a8ddd64d763f8788bec75491feR31), but you forgot to require it from the [solidus_bolt.rb](https://github.com/nebulab/solidus_bolt/blob/master/lib/solidus_bolt.rb#L5) file.

Reply:
added require to `solidus_bolt.rb`